tests/test_getmud.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from diffpy.labpdfproc.getmud import estimate_diameter, estimate_mass_density
+
+
+@pytest.mark.parametrize(
+    "inputs, expected_mass_density",
+    [
+        (
+            {
+                "mud": 2.0,
+                "diameter": 1.5,
+                "sample_composition": "ZrO2",
+                "energy": 20,
```

Comment:
it is probably a better test if we use reasonable energies.  These would be Cu Kalpha, Mo Kalpha, Ag Kalpha.  They don't have to be exact but ballpark,